articlestatus/ArticleStatusHelper.js

```diff
@@ -0,0 +1,27 @@
+const ArticleStatus = require('../data/models/ArticleStatus');
+
+class ArticleStatusHelper {
+  async storeArticleStatus(userid, articleid, offset) {
+    const astat = new ArticleStatus({
+      pocket_user_id: userid,
+      article_id: articleid,
+      offset_ms: offset
+    });
+    return await astat.save();
+  }
+
+  async getArticleStatus(userid, articleid = -1) {
+    if (articleid == -1) {
```

Comment:
Can clean this up a bit by dropping the default value
```
  async getArticleStatus(userid, articleid) {
    if (!articleid) {
```

Reply:
Oh didn't know we could do that. Thanks!